.github/workflows/test.yml

```diff
@@ -100,7 +100,7 @@ jobs:
           bundle install --jobs 4 --retry 3
         working-directory: ./ios
 
-      - run: pod install --repo-update
```

Comment:
undo this change. was it necessary?

Reply:
hm, are you sure we want to pass `repo-update`? It will pull down the latest available version if the Podfile.lock does not specify otherwise